crates/sui-json/src/tests.rs

```diff
@@ -21,6 +21,7 @@ use sui_types::base_types::{
     ObjectID, SuiAddress, TransactionDigest, STD_ASCII_MODULE_NAME, STD_ASCII_STRUCT_NAME,
     STD_OPTION_MODULE_NAME, STD_OPTION_STRUCT_NAME,
 };
+use sui_types::gas_coin::GasCoin;
```

Comment:
Dont mean to have to block this but if possible please add a test for homogeneity and non-homogeneity
Example https://github.com/MystenLabs/sui/blob/6c5c239ff980ef48b18297c00460f41ea19e4930/crates/sui-json/src/tests.rs#L33